### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**
## Summary

Adds documentation on various subjects.

I have created a root-level `docs` directly because scattering the docs over the various README files of packages didn't seem the most practical. For example the docs on lokalize texts were part of the cms package, but the concept is very relevant for other parts of the codebase too.

### CODE REVIEW

File: docs/lokalize-texts.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+The `sync-after-feature` command is triggered automatically by a Github Action
+whenever a feature branch is merged to the develop branch. It contains the
+following logic:
+
+1. Apply moves and deletions to the development set. This will break other
+   feature branches, but at this point those branches can be updated with the
+   develop new commits.
+2. Sync additions to the production set. Any text that was added as part of this
+   feature branch is added to the production set so that the communication team
+   can prepare the texts for the upcoming release. These text get a special flag
```
Comment (VWSCoronaDashboard1) [inline docs/lokalize-texts.md:142]:
```suggestion
   can prepare the texts for the upcoming release. These texts get a special flag
```

File: docs/lokalize-texts.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+   through wrong use of the mutations file or something.
+
+After these steps the production dataset should be considered a mirror of the
+development dataset.
+
+It is possible that right after a release there are already new texts in the
+development set which are part of the next sprint. Those will get added to
+production as well but this won't be much of a problem.
+
+**DANGER:** One thing we need to keep in mind is to not delete or move texts
```
Comment (VWSCoronaDashboard1) [inline docs/lokalize-texts.md:173]:
```suggestion
**DANGER:** One thing we need to keep in mind is not to delete or move texts
```

File: docs/release-procedure.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+9. Once ready, merge the release branch to `master` using a **merge commit.
+   Never use a squash and merge action, since this will erase/rewrite the commit
+   history.**
+10. Hit the publish button in the Github release draft. This should tag the
+    correct commit in `master`
+
+## CMS Deploy
+
+Sometimes a release also contains new configurations for the CMS. These are
+deployed via the Sanity CLI as `yarn run sanity deploy`. These deployments do
```
Comment (VWSCoronaDashboard1) [inline docs/release-procedure.md:65]:
```suggestion
deployed via the Sanity CLI as `yarn deploy`. These deployments do
```

File: docs/timeline-events.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Timeline Events
+
+All time-series charts can optionally render events on a timeline to illustrate
+and describe certain parts of the data.
```
Comment (VWSCoronaDashboard1) [inline docs/timeline-events.md:4]:
```suggestion
and describe certain points or ranges of the data.
```

File: docs/timeline-events.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+As an example, the chart that is shown on the safety region page about hospital
+admissions exists in the CMS as an element with id
+`vr__hospital_nice__time_series`, and for the national page on confirmed cases
+we have an element with id `nl__tested_overall__time_series`.
+
+## Adding An Element
+
+Adding a new element is only possible via a command-line interface.
+
+@TODO
```
Comment (VWSCoronaDashboard1) [inline docs/timeline-events.md:32]: Is this really a TODO? Or did you forget to make this part of this PR?

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: docs/timeline-events.md (hunk 5 above)
Comment (VWSCoronaDashboard8) [reply docs/timeline-events.md:32]: The CLI doesn't exist yet, so it is a todo for after this PR

**Pull request merged**